components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -642,6 +642,18 @@ def leave_none_unset_int(s):
          ("Whether or not to include a user dropdown in the base template."
           " Particularly useful when used in combination with the OMERO.web"
           " public user where logging in may not make sense.")],
+    "omero.web.feedback.comment.enabled":
+        ["FEEDBACK_COMMENT_ENABLE",
+         "true",
+         parse_boolean,
+         ("Enable the feedback form for comments. "
+          "These comments are sent directly to the OME team.")],
```

Comment:
Not necessarily the case. Maybe something along the lines of "to the URL specified by `omero.qa.feedback` (OME team by default)"?